lib/style_parser/namespacer/prefixer.js

```diff
@@ -24,15 +24,20 @@ function prepend(styles, prefix, cb) {
 }
 
 function strip(styles, prefix, cb) {
-
   var obj = css.parse(styles);
   var sheet = obj.stylesheet;
   var outputCache = [];
   var selectors = [];
   var rules = [];
+  var isReverted = false;
```

Comment:
isReverted is used to control the repeating loop , which needs to call prefixer or postfixer until all the prefixes or postfixes are removed.
So it goes to prefixer just one extra time, this time it doesn't find any prefixes and return without reverting anything. And the loop doesn't continue if isReverted is returned false.